objcgen/processor.cs

```diff
@@ -67,7 +67,7 @@ public void Process (ProcessedAssembly a)
 
 		protected void AddExtraType (ProcessedType pt)
 		{
-			typeQueue.Enqueue (pt);
+			Types.Add (pt);
```

Comment:
No, I mean always create the `typeQueue` (like `assemblyQueue`). That will make `AddExtraType` work. It also requires to enqueue all items in `Types` before clearing it.

Ideally `AddExtraType` should always be callable, since there can more than than one (current) reason to use it.